astro_2.0/components/DaoDetails/DaoDetailsGrid.module.scss

```diff
@@ -76,12 +76,31 @@
 .name {
   @include font(28px, 28px, 700);
   font-family: 'Roboto', 'sans-serif';
+
+  @include astro-mobile {
+    font-size: 20px;
+    > div {
+      @include ellipse-text;
+      max-width: 180px;
+    }
+  }
 }
 
 .address {
   @include font(16px, 24px, 500);
+  align-items: center;
   color: var(--color-neutral-60);
+  display: flex;
+  margin-top: 8px;
   word-break: break-all;
+
+  @include astro-mobile {
+    font-size: 14px;
+    > div {
+      @include ellipse-text;
+      max-width: 180px;
+    }
```

Comment:
It's quite "dangerous" to tighten styles to the structure. Maybe, we can achieve same by providing classNames to nested components?